ivy/functional/frontends/jax/devicearray.py

```diff
@@ -23,6 +23,8 @@ def at(self):
     # Instance Methods #
     # ---------------- #
 
+    def __all__(self, other):
```

Comment:
@FaikYY thanks for your contribution! Could you please remove the changes to the files in `.idea` folder from your commit? Also, would be great if you could also write a test for your implementation. You can have a look at the [docs ](https://lets-unify.ai/ivy/deep_dive/ivy_frontends_tests.html) and the tests for other  [jax instance methods](https://github.com/unifyai/ivy/blob/master/ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py) to get an idea of how to test your function.

Reply:
Thanks a lot for your feedback! I will remove the changes from .idea file and write a test.